menas/pom.xml

```diff
@@ -109,11 +109,6 @@
             <artifactId>mongo-scala-driver_${scala.compat.version}</artifactId>
             <version>${mongo.scala.driver.version}</version>
         </dependency>
-        <dependency>
```

Comment:
The same dependency is in `migrations`. Maybe remove it from these as well?

Reply:
Smart! I might have merged without doing this if the reviews didn't go stale.